exp-player/addon/components/exp-rating-form/component.js

```diff
@@ -0,0 +1,560 @@
+      'measures.questions.10.options.neutral',
+      'measures.questions.10.options.agree',
+      'measures.questions.10.options.agreeStrongly'
+    ],
+    {labelTop: true}
+  ),
+  {
+    question: 'measures.questions.11.label',
+    type: 'radio-input',
+    scale: ['measures.questions.11.options.yes', 'measures.questions.11.options.no'],
+    items: {
+      radio: {
+        value: null
+      },
+      input: {
+        description: 'measures.questions.12.label',
+        value:null,
+        optional:true
+      }
+    }
+  },
+  generateSchema(
+    'measures.questions.13.label',
+    'radio',
+    items['13'],
+    [
+      'measures.questions.13.options.disagreeStrongly',
+      'measures.questions.13.options.disagree',
+      'measures.questions.13.options.neutral',
+      'measures.questions.13.options.agree',
+      'measures.questions.13.options.agreeStrongly'
+    ],
+    {labelTop: true}
+  ),
+  generateSchema(
+    'measures.questions.14.label',
+    'radio',
+    items['14'],
+    [
+      'measures.questions.14.options.disagreeStrongly',
+      'measures.questions.14.options.disagree',
+      'measures.questions.14.options.neutral',
+      'measures.questions.14.options.agree',
+      'measures.questions.14.options.agreeStrongly'
+    ],
+    {labelTop: true}
+  ),
+  generateSchema(
+    'measures.questions.15.label',
+    'radio',
+    items['15'],
+    [
+      'measures.questions.15.options.disagreeStrongly',
+      'measures.questions.15.options.disagree',
+      'measures.questions.15.options.neutral',
+      'measures.questions.15.options.agree',
+      'measures.questions.15.options.agreeStrongly'
+    ],
+    {labelTop: true}
+  ),
+  generateSchema(
+    'measures.questions.16.label',
+    'radio',
+    items['16'],
+    [
+      'measures.questions.16.options.notAtAll',
+      'measures.questions.16.options.aLittle',
+      'measures.questions.16.options.quiteaBit',
+      'measures.questions.16.options.completely'
+    ],
+    {labelTop: true}
+  ),
+  generateSchema(
+    'measures.questions.17.label',
+    'radio',
+    items['17'],
+    [
+      'measures.questions.17.options.disagreeStrongly',
+      'measures.questions.17.options.disagree',
+      'measures.questions.17.options.neutral',
+      'measures.questions.17.options.agree',
+      'measures.questions.17.options.agreeStrongly'
+    ],
+    {labelTop: true}
+  )
+];
+
+const Validations = buildValidations(generateValidators(questions));
+
+export default ExpFrameBaseComponent.extend(Validations, {
+    type: 'exp-rating-form',
+    layout: layout,
+    meta: {
+        name: 'ExpRatingForm',
+        description: 'TODO: a description of this frame goes here.',
+        parameters: {
+            type: 'object',
+            properties: {
+                // define parameters here
+            }
+        },
+        data: {
+            type: 'object',
+            properties: {
+                questions: {
+                    default: questions
```

Comment:
Having this refer to a `questions` defined outside of the class instance feels risky to me. 

`data` defines what data is frame will export to be saved. Maybe it makes more sense to add a computed to fetch the actual values of the questions (minus some of the extra boilerplate?)?